index.html

```diff
@@ -20,63 +20,197 @@
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
+          </article>
+          <article class="card">
+            <h3 class="heading">Profesional Art Printing Data</h3>
+            <p class="blurb">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="badges">
+              <li class="badge">HTML</li>
+              <li class="badge">Bootstrap</li>
+              <li class="badge">Ruby</li>
+            </ul>
+            <button type="button" class="btn">See project</button>
```

Comment:
- [ ] Please make sure that all buttons have their interactions styles set up, to better match the Figma template.

![image](https://github.com/Drod86/My_Portfolio/assets/62290545/f2abb712-1853-486a-b434-e515a0194dd9)